### DISCUSSION TITLE
Silent Bad Request & Internal Server Errors

### DISCUSSION THREAD

**Discussion by deslashcode** [standardnotes/forum#2467] [General] [answered]
**Describe the bug**
Scanning the open ports shows mulitple "Bad Requests" and "Internal Server Errors" on port 45653 with the error message "Unable to load the extension. Please restart the application or try reinstalling the application". The application itself does not explicitly show or indicate any such error for any extension for the user to take any action.

**To Reproduce**
1. nmap -A 127.0.0.1 -p 45653
2. lsof -Pnl +M -i4
3. Step 1 gives the service fingerprint with detailed error message

**Expected behavior**

1) If there's an extension failing to communicate with the server, the application should inform the user explicitly with an error pop-up or indication listing the extension name and the reason it was communicating with the server and why the failure's happening.
If this error message is not accurate or related to the cause, an appropriate message with specific details will help.

2) Service mapped to the port shows up as "unkown service". If the port is indeed correct, registering the service fingerprint will make it easier to identify.

**Screenshots**
![sn2](https://user-images.githubusercontent.com/65415242/89725154-d9659f80-da0c-11ea-89e6-811e016a340e.png)
![sn1](https://user-images.githubusercontent.com/65415242/89725153-d66aaf00-da0c-11ea-8929-e842a2c3033e.png)

**Desktop (please complete the following information):**
 - Linux

**Comment by atmoio** [comment]
Port 45653 is for the local ext server the desktop app uses to host local extensions. If you give it an invalid or root path, you'll get an error. Seems like this is the correct result.

**Comment by deslashcode** [comment]
I am not sure I understand the 'giving it an invalid/root path' part there. The error console points to "Plus Editor". Uninstalled and reinstalled it. What path are we talking about here? To be clear, the editor (and the app) loads fine despite this.

**Comment by myreli** [comment, marked as answer]
We are triaging our old issues and this does not seem to happen any more on `3.149.5`. Let us know if you have any problems.
